Source/WebKit/WPEPlatform/wpe/drm/WPEDisplayDRM.cpp

```diff
@@ -375,8 +375,10 @@ static gboolean wpeDisplayDRMSetup(WPEDisplayDRM* displayDRM, const char* device
         auto scale = parseDouble(trimmedScaleString, parsedLength);
         if (parsedLength == trimmedScaleString.length() && scaleIsInBounds(scale))
             scaleFromEnvironment = scale;
-        else
-            g_warning("Invalid WPE_DRM_SCALE='%*s' value, or out of bounds.", static_cast<int>(scaleString.span8().size()), scaleString.span8().data());
+        else {
+            CString cstring = scaleString.utf8();
+            g_warning("Invalid WPE_DRM_SCALE='%*s' value, or out of bounds.", static_cast<int>(scaleString.size()), scaleString.data());
```

Comment:
How does this work? One format parameter, two arguments? I'm not sure what `%*s` does but I'm pretty tempted to rewrite this.

Reply:
OK, it doesn't work. Already fixed in [300311@main](https://commits.webkit.org/300311@main), but got reverted in [300340@main](https://commits.webkit.org/300340@main) and we failed to reland it.